1-exercises/B-array-some/exercise.js

```diff
@@ -11,14 +11,22 @@ let pairsByIndex = [[0, 3], [1, 2], [2, 1], null, [3, 0]];
 // If there is a null value in the array exit the program with the error code
 // https://nodejs.org/api/process.html#process_process_exit_code
 // process.exit(1);
+ function isNull(element) {
+   return element === null;
+ }
```

Comment:
good choice of naming convention; functions that return booleans often are named `is...`